explainers/hidden-content-explainer.md

```diff
@@ -271,15 +233,83 @@ only benefit I see to adding support to `window.find` would be to make it easier
 to add WPT tests for beforematch, since we cant have WPTs for find-in-page.
 However, that would also require speccing `window.find`.
 
+## Backwards compatibility with the `hidden` attribute
+
+If a browser which doesn't implement this feature renders a `hidden=until-found`
+element, it will still apply the `display:none` style from the `hidden`
+attribute, which will still make the element hidden like the
+`content-visiblity:hidden` style that `hidden=until-found` is supposed to apply.
+This means that hopefully websites won't need to specially consider support for
+this feature between browsers.
+
+However, `content-visibility:hidden` will still allow some parts of the
+`hidden=until-found` element to be rendered: the [generated
+box](https://drafts.csswg.org/css2/#visibility) of the element.
+
+As a mitigation for this, the following CSS should be included in the user agent
+stylesheet:
+```css
+[hidden=until-found] {
+  visibility: hidden;
+}
+[hidden=until-found] > * {
+  visibility: visible;
```

Comment:
I forgot what if we said that things like
```html
<div style="visibility: hidden">
  <div hidden=until-found>text</div>
</div>
```
would be searchable with this style

Reply:
ill discuss this some more in the design doc